data-raw/northern-ireland-ltla-summary-metrics.R

```diff
@@ -0,0 +1,129 @@
+library(tidyverse)
+library(IMD)
+library(geographr)
+library(compositr)
+library(sf)
+library(ggridges)
+
+ltla <-
+  boundaries_ltla21 |>
+  st_drop_geometry() |>
+  filter(str_detect(ltla21_code, "^N"))
+
+lookup_northern_ireland_ltla <-
+  lookup_ltla_ltla |>
+  filter(str_detect(ltla21_code, "^N"))
+
+# ---- IMD score ----
+# Higher extent = more deprived /
+# Higher rank (calculated here) = more deprived
+imd <-
+  imd_northern_ireland_lad |>
+  select(ltla21_code = lad_code, imd_score = Extent) |>
+  mutate(number = rank(imd_score)) |>
+  select(-imd_score) |>
+  mutate(variable = "Index of Multiple \nDeprivation rank", .after = ltla21_code) |>
+  mutate(percent = NA, .after = number)
+
+# ---- ONS Health Index ----
+# An official Health Index for Northern Ireland does not exists. Use the BRC Resilience Index version
+# Higher score = worse health
+# Higher rank (calculated here) = worse health
+health_index_raw <- read_csv(
+  "https://raw.githubusercontent.com/britishredcrosssociety/resilience-index/main/data/vulnerability/health-inequalities/northern-ireland/index-unweighted-all-indicators.csv"
+)
+
+health_index <-
+  health_index_raw |>
+  select(ltla21_code = lad_code, number = health_inequalities_composite_rank) |>
+  mutate(percent = NA) |>
+  mutate(variable = "Health Index \nrank") |>
+  relocate(variable, .after = ltla21_code)
+
+# ---- % Left-behind areas ----
+# The left-behind areas come from the community needs index (loaded from the IMD package)
+# Higher number/percent = more left-behind
+lba <-
+  cni_northern_ireland_soa11 |>
+  left_join(lookup_northern_ireland_ltla, by = c("lgd14_code" = "ltla19_code")) |>
```

Comment:
Please can we remove this lookup and just rename `lgd14_code` in the select statement below:

```r
select(soa11_code, ltla21_code = lgd14_code, lba = `Left Behind Area?`)
```

Reply:
Please wrap this line at 80 characters:

```r
left_join(
    lookup_northern_ireland_ltla,
    by = c("lgd14_code" = "ltla19_code")
  ) 
```